packages/semantic-ui/src/TextareaWidget/TextareaWidget.js

```diff
@@ -24,7 +23,7 @@ function TextareaWidget(props) {
     uiSchema,
     formContext,
   } = props;
-  const semanticProps = getSemanticProps({ formContext, options });
+  const semanticProps = getSemanticProps({ formContext, options, defaultSchemaProps: {} });
```

Comment:
Given the example from `EmailWidget` should `defaultSchemaProps` be assigned as the following since the defaultProps had that?:
```
    defaultSchemaProps:{
      inverted: false,
    }
```